### PR TITLE
fix: revert the hack for the CLI integration tests

### PR DESCRIPTION
**Pull request by hansl**
Revert #21611.

### CODE REVIEW

**Review by alexeagle — APPROVED** [review]

**Review by mhevery — APPROVED** [review]

File: aio/tools/examples/shared/package.json [hunk 1]
```diff
@@ -43,7 +43,7 @@
     "zone.js": "^0.8.4"
   },
   "devDependencies": {
-    "@angular/cli": "1.5.0",
+    "@angular/cli": "1.6.5",
```
Comment (gkalpak) [inline aio/tools/examples/shared/package.json]: You need to also update the `yarn.lock` file in this dir (or Travis complains :smiley:)

**Comment by petebacondarwin** [comment]
It would be simpler, I think to land https://github.com/angular/angular/pull/21772 first.

**Comment by hansl** [comment]
Thanks @petebacondarwin.  So many `package.json` and `yarn.lock` I just lost track :D

**Comment by petebacondarwin** [comment]
@hansl - I think that this PR is still valid, since it removes the "hack", which my PR did not touch.

**Pull request closed without merging**